currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/BitcoinFragment.kt

```diff
@@ -156,10 +232,22 @@ class BitcoinFragment : BaseFragment(R.layout.fragment_bitcoin),
         fun newInstance() = BitcoinFragment()
     }
 
-    override fun onImport(address: String, privateKey: String, testNet: Boolean) {
+    override fun onImport(address: String, privateKey: String, testNet: Int) {
         if (!WalletManagerAndroid.isRunning) {
             val config = WalletManagerConfiguration(
-                if (testNet) BitcoinNetworkOptions.TEST_NET else BitcoinNetworkOptions.PRODUCTION,
+                when (bitcoin_network_radio_group.checkedRadioButtonId) {
+                    1 -> BitcoinNetworkOptions.PRODUCTION
+                    2 -> BitcoinNetworkOptions.TEST_NET
+                    3 -> BitcoinNetworkOptions.REG_TEST
+                    else -> {
```

Comment:
I would just select one of them by default and then you can drop the else clause

Reply:
Unfortunately, when I remove the else branch, Kotlin starts yelling that my when expression should be exhaustive and the 'else' branch is missing... 
But I will make the RegTest default anyway